### PR TITLE
gh4: Resolving syntax error

### PR DESCRIPTION
**Pull request by tulips8**
- [ ] Code changes made to support the switch in the correct place.
- [ ] Upgrade TODO to state that switches should not be used in the future
- [ ] The file must be tested as building with the rest of the stack.

### CODE REVIEW

File: include/ccsp_hal_ethsw.h [hunk 1]
```diff
@@ -540,7 +540,7 @@ INT CcspHalExtSw_getEthWanEnable(BOOLEAN *pFlag);
  * @retval RETURN_ERR - On failure.
  */
 INT CcspHalExtSw_setEthWanEnable(BOOLEAN Flag);
-
+#ifdef FEATURE_RDKB_AUTO_PORT_SWITCH
```
Comment (Ulrond) [inline include/ccsp_hal_ethsw.h:543]: interfaces should not have #ifdef switches in them, the interface is fixed for all platforms.

Comment (tulips8) [reply include/ccsp_hal_ethsw.h:543]: We cannot change the code, can we add a TODO?

Comment (Ulrond) [reply include/ccsp_hal_ethsw.h:543]: yes do that.

**Review by Ulrond — APPROVED** [review]
looks good

- [x] Code changes made to support the switch in the correct place.
- [x] Upgrade TODO to state that switches should not be used in the future

Can you confirm the 3rd Point

- [ ] The file must be tested as building with the rest of the stack.

**Review by Ulrond — APPROVED** [review]

**Review by Ulrond — APPROVED** [review]

**Pull request merged**